hc/accounts/tests/test_check_token.py

```diff
@@ -22,5 +22,8 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+        assert r.status_code == 302
 
-    ### Login with a bad token and check that it redirects
\ No newline at end of file
+    ### Login with a bad token and check that it redirects
+        c = self.client.post("/accounts/check_token/rose/secret-token/")
+        assert c.status_code == 302
```

Comment:
Something about this assertion isn't quite right. Have a look at Django's documentation on assert statements [here](https://goo.gl/Ra6s5u)